src/devices/Card/CreditCard/TagList.cs

```diff
@@ -0,0 +1,307 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace Iot.Device.Card.CreditCardProcessing
+{
+    /// <summary>
+    /// Class containing the tag list definitions, their types, sources and numbers
+    /// </summary>
+    public class TagList
+    {
+        /// <summary>
+        /// List of Tags description, source, conversion for display as well as if they are templates or contain Data Object Link
+        /// This list is coming from the EMV documentation, see EMV book 3, Annexe A
+        /// </summary>
+        public static List<TagDetails> Tags = new List<TagDetails>()
+        {
+            new TagDetails() { TagNumber =0x06, Source = Source.Icc, Decoder = ConversionType.ByteArray, Description="Object Identifier (OID)", IsTemplate = false, IsDol = false },
```

Comment:
nit: space after `=` for all here

Reply:
285 global replace done. I adjusted as well the one for Description (was missing before and after)

Reply:
And found as well one with "'T" which I fixed as well